.github/workflows/pull_request.yml

```
@@ -0,0 +1,84 @@
+# Copyright 2018-2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: CI
+
+on: ['pull_request']
```

Should this be tweaked to be on pull requests that target the master branch? Otherwise, you'll end up having PRs into existing PRs that will cause this to trigger.

```suggestion
on:
  pull_request:
    branches:
      - master
```

---

We need this also on PRs to other release branches also, like  `v0.6.x` etc 

> you'll end up having PRs into existing PRs that will cause this to trigger.

Which is this case. I didn't get it.

---

You can do a list with regex. EG.
```
- master
- v*
```

The scenario is as follows:

base: master <-- source: feature-1         (This will trigger workflow)
base: feature-1 <-- source: feature-2     (This will trigger the workflow, but it probably shouldn't?)

---

Done.